pywr-schema/src/outputs/csv.rs

```diff
@@ -65,9 +117,11 @@ mod tests {
 
         model.run::<ClpSolver>(&ClpSolverSettings::default()).unwrap();
 
-        // After model run there should be an output file.
-        let expected_path = temp_dir.path().join("outputs.csv");
-        assert!(expected_path.exists());
+        // After model run there should be two output files.
+        let expected_long_path = temp_dir.path().join("outputs-long.csv");
+        assert!(expected_long_path.exists());
+        let expected_wide_path = temp_dir.path().join("outputs-wide.csv");
+        assert!(expected_wide_path.exists());
```

Comment:
Would it be worth loading the outputs csv in these tests and doing some basic checks (e.g. shape, headers, etc..)?

Reply:
Done.